backend/src/blueprints/messages/messages_views.py

```diff
@@ -0,0 +1,30 @@
+from flask import Blueprint
+
+from src.blueprints.messages.messages_service import (
+    get_public_message,
+    get_protected_message,
+    get_admin_message,
+)
+from src.blueprints.security.guards import (
+    authorization_guard,
+)
+
+bp_url_prefix = "/messages"
+messages = Blueprint("messages", __name__, url_prefix=bp_url_prefix)
+
+
+@messages.route("/public")
+def public():
+    return vars(get_public_message())
```

Comment:
this is an interesting way of going about json serialization. lgtm for now, but we'll probably need to discuss this later.